django/db/models/base.py

```diff
@@ -324,7 +326,7 @@ def _prepare(cls):
 
             # defer creating accessors on the foreign class until we are
             # certain it has been created
-            def make_foreign_order_accessors(field, model, cls):
+            def make_foreign_order_accessors(cls, model, field):
```

Comment:
I don't understand this response - *this* API doesn't take `n` models, even in the new version, so why should its parameter ordering change?